routes/api/gameApi.js

```diff
@@ -19,13 +37,13 @@ router.post("/", withAuth, async (req, res) => {
       });
 
       res.json(game);
-    } catch (e) {
-      console.log(e);
+    } catch (err) {
+      console.log(err);
```

Comment:
remove

Reply:
I will remove that fully as belong to different branch